README.md

```diff
@@ -103,6 +107,10 @@ regarding this example. The quickstart guide also contains an example of how to
 
 ## Examples and Tutorials
 
+Check out [NVIDIA LaunchPad](https://www.nvidia.com/en-us/data-center/products/ai-enterprise-suite/trial/)
```

Comment:
Remove trailing "/"

Reply:
This file is cherry-picked from `main` branch and it's a part of different commit, can cause a merge conflicts later.